packages/devtools_app/lib/src/standalone_ui/vs_code/devices.dart

```diff
@@ -12,15 +12,21 @@ import '../../shared/analytics/constants.dart' as gac;
 import '../api/vs_code_api.dart';
 
 class Devices extends StatelessWidget {
-  const Devices(
-    this.api,
-    this.devices, {
+  Devices(
+    this.api, {
+    required this.devices,
+    required this.unsupportedDevices,
     required this.selectedDeviceId,
     super.key,
-  });
+  }) : unsupportedDevicePlatformTypes = unsupportedDevices
+            .map((device) => device.platformType)
+            .nonNulls
+            .toSet();
 
   final VsCodeApi api;
   final List<VsCodeDevice> devices;
+  final List<VsCodeDevice> unsupportedDevices;
```

Comment:
nit: what about a single list of devices with a boolean field 'supported' so that we don't have to maintain two lists?

Reply:
It'll be slightly more complicated because we'd need DevTools to tell VS Code that it knows to check the flag (otherwise, new VS Code would include unsupported devices to old DevTools, and users would see "broken" devices in the list because the old Sidebar didn't filter them out).

We don't currently have capabilities in this direction (for VS Code to know what DevTools supports), but adding support for it might not be a bad idea (we're probably going to want it in future for other things).

(we could also base it on an SDK version number, but that's a little more awkward because we'd have to land the DevTools changes before we have a version number to code into VS Code)

Reply:
Gotcha. Keeping two lists sgtm then.